test/envtest/setup.go

```diff
@@ -82,3 +82,18 @@ func Setup(t *testing.T, scheme *runtime.Scheme) *rest.Config {
 
 	return cfg
 }
+
+func InstallKongCRDs(t *testing.T, cfg *rest.Config) {
+	projectRoot := os.Getenv("KONG_TEST_ENVTEST_PROJECT_ROOT")
```

Comment:
I think we should try to not introduce unnecessary test configs - I bumped into the same issue with installing Kong CRDs in my PR and found a solution that allows you to build an absolute path to the CRDs' directory based on the file runtime metadata, PTAL:

https://github.com/Kong/kubernetes-ingress-controller/pull/4409/files#diff-27c645594dcd9d17888cf588882ed1a2e724b27de898e2b70257cd3e9fbfc247R49

Reply:
Thanks for providing the `options` to configure envtest. I did some modification on your setup there, and added an `OptionModifier` to allow more flexibility and less touch on existing parts.

Reply:
Yeah, I like it this way! 🎖️